### ISSUE TITLE
After dismissing a notification; the notification-badge-link remains with no content

### ISSUE THREAD

**Issue by SjonHortensius** [Graylog2/graylog2-web-interface#1003]
After dismissing a single notification, the notification count is decreased but the badge isn't hidden; the remaining HTML is:

```html
<li>
   <a href="/system?page=0" class="notification-badge-link">
      <span class="badge badge-important" id="notification-badge" style="display: inline-block; opacity: 0.616722681927953;"></span>
   </a>
</li>
```

**Comment by edmundoa** [comment]
Hi,

I could not reproduce this issue, the notification badge takes a few seconds to update itself, but when there are no notifications it's hidden for me. Which Graylog version are you using?

**Comment by SjonHortensius** [comment]
Try opening 2 tabs; dismiss the notification in one tab, the other tab will show an empty span. I use v0.92.3

**Comment by edmundoa** [comment]
I just tried with the current 1.0 branch and a bunch of opened tabs in Firefox and Chrome and I couldn't reproduce the issue. There is a hidden empty `span` in the DOM, but I can't see anything or click anywhere.

As this seems to be fixed in the code that will be part of 1.0, I think the best is to close the issue for now. Please reopen if it happens again in future versions.
